osquery/tables/system/linux/startup_items.cpp

```diff
@@ -100,6 +165,12 @@ QueryData genStartupItems(QueryContext& context) {
     genAutoStartItems(dir, results);
```

Comment:
When run in `--verbose`, the `VLOG()` calls on line 32 (in `genAutoStartIems()`) and 69 (`genSystemdItems()`) generate about 100 "error" messages about non-existent directories.

If it's necessary to check for all of those, but expected that many of them won't exist, then maybe the output could be suppressed.

Reply:
This function is not part of the patch, we should probably open a new PR